README.md

```diff
@@ -15,8 +15,8 @@ extend Fission if you want to.
 
 Fission is extensible to any language; the core is written in Go, and
 language-specific parts are isolated in something called
-_environments_ (more below).  Fission currently supports NodeJS and
-Python, with more languages coming soon.
+_environments_ (more below).  Fission currently supports NodeJS, .NET, Ruby, Bash
+PHP, Python, Go, and any Linux-compiled binary, with more languages coming soon.
```

Comment:
Thanks. Can we use this order: NodeJS, Python, Ruby, Go, PHP, Bash, and any Linux executable.